### DISCUSSION TITLE
Round Cornered Window

### DISCUSSION THREAD

**Discussion by sekkit** [AvaloniaUI/Avalonia#10444] [Q&A]
How to make window roundcornered?
on Win11, window is round cornered, but on Win10, it's rectangle.

**Comment by timunie** [comment]
I think this is a "feature" (I personally don't like it) shipped with Windows 11. This is not only the case for Avalonia Apps. Almost all programs I use by a daily base have rounded corners on Win 11 and sharp edges. 

If you really need or want to change the corner radius for all OS, you need to remove the system decorations and re-implement them on your own. But this is probably not an easy task and may not work on Linux. 

**Example:** 

```xml
<Window xmlns="https://github.com/avaloniaui"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
        mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
        x:Class="Test.MainWindow"
        TransparencyLevelHint="Transparent"
        Background="Transparent"
        SystemDecorations="None"
        Width="150" Height="100"
        Title="Test">

	<Border CornerRadius="20" Background="DarkBlue" >
		<TextBlock Text="This is an example"
		           HorizontalAlignment="Center"
		           VerticalAlignment="Center"/>
	</Border>
</Window>
```

Result:
![image](https://user-images.githubusercontent.com/47110241/220858388-73fe0483-cb81-472c-a859-65e174593514.png)


**Note:** I'll transfer this issue into Q&A as it is not a bug or feature we can add to Avalonia. I hope you agree.
